app/views/layouts/application.html.erb

```diff
@@ -339,5 +339,7 @@
       </script>
 
     <%= javascript_include_tag 'application' %>
+
+    
     </body>
```

Comment:
since we didn't really change anything in this file, can we remove it from the PR